WalletWasabi/Stores/BitcoinStore.cs

```diff
@@ -31,6 +35,7 @@ public class BitcoinStore
 		public AllTransactionStore TransactionStore { get; }
 		public SmartHeaderChain SmartHeaderChain => IndexStore.SmartHeaderChain;
 		public MempoolService MempoolService { get; }
+		public IRepository<uint256, Block> Blocks { get; }
```

Comment:
`Blocks` is name that denotes rather a collection of blocks but not something that provides you blocks. I would rather name it `BlockRepository` that it what it is.